proto/rill/runtime/v1/api.proto

```diff
@@ -215,6 +219,24 @@ message PingResponse {
   google.protobuf.Timestamp time = 2;
 }
 
+message HealthzRequest {}
```

Comment:
Wondering if we should either:
- Add support for an optional `instance_id` here to limit checks to only one instance
- Or extract instance checks to a separate `InstanceHealth` RPC

Thinking we might want to expose the instance health check in `rill project status` as well.

Reply:
Extracting to `InstanceHealth` seems better. Otherwise need to ping registry and other common stores for every instance.